lib/_debug_agent.js

```diff
@@ -20,7 +20,14 @@ exports.start = function start() {
 
   agent.listen(process._debugAPI.port, function() {
     var addr = this.address();
-    process._rawDebug('Debugger listening on port %d', addr.port);
+    // SmartOS sometimes munges output from `process._rawDebug()`
+    // but Windows can't use `console.error()`.
+    // https://github.com/nodejs/node/issues/2476
+    if (process.platform == 'win32') {
+      process._rawDebug('Debugger listening on port %d', addr.port);
+    } else {
+      console.error('Debugger listening on port %d', addr.port);
```

Comment:
Oh, yeah, meant to assign to a function before submitting PR. Will change. Template string is a great idea. Didn't even occur to me.